UM/Qt/QtApplication.py

```diff
@@ -598,10 +599,38 @@ def closeSplash(self) -> None:
             QtApplication.splash.close()
             QtApplication.splash = None
 
+    def _onWindowVisibleChange(self, visible: bool) -> None:
+        if not visible:
+            self._sub_windows.remove(self.sender())
+
+    def createQmlSubWindow(self, qml_file_path: str, context_properties: Dict[str, "QObject"] = None) -> Optional["QQuickWindow"]:
+        """
+        Create a QML window from a QML file. This is different from createQmlComponent in 3 different ways:
```

Comment:
Since you're still using that method _within_ this one, it's a bit confusing (though technically correct) to say that it's 'different' -- maybe just state that this is a wrapper that handles some additional tasks instead.